src/app/public/modules/data-manager/data-manager.component.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+
+import {
+  Subject
+} from 'rxjs/Subject';
+
+import {
+  SkyUIConfigService
+} from '@skyux/core';
+
+import {
+  SkyDataManagerService
+} from './data-manager.service';
+
+import {
+  SkyDataManagerConfig,
+  SkyDataManagerState,
+  SkyDataManagerStateOptions
+} from './models';
+
+@Component({
+  selector: 'sky-data-manager',
+  templateUrl: './data-manager.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyDataManagerComponent implements OnDestroy, OnInit {
+
+  @Input()
+  public set activeViewId(value: string) {
+    this.dataManagerService.updateActiveViewId(value);
+  }
+
+  @Input()
+  public dataManagerConfig: SkyDataManagerConfig;
```

Comment:
Do we expect consumers to push changes the config and have the data manager react to those changes? Or is this a one-time thing the consumer will provide on init?

Reply:
hmm good question. It is technically an updatable thing via the data manager service, though I don't anticipate it will be frequently updated. The data manager would react to changes to it by subscribing to the observable, the toolbar currently does that since it needs to keep the toolbar options up to date. Right now consumers pass their config and the data manager updates it in the service, do you think it would be better to have the consumers just set it via the service? @Blackbaud-AlexKingman

Reply:
Hmm... I'm not sure which approach is best here. At the least, I feel like it should be able to react to changes if the consumer pushes a new `SkyDataManagerConfig` object. We may want to discuss this in a larger group?